crates/ingress-rpc/src/service.rs

```diff
@@ -218,6 +225,21 @@ where
 
         Ok(transaction.tx_hash())
     }
+
+    async fn send_user_operation(
```

Comment:
i'm not too familiar with UserOps, so I'd be curious to know: who are the callers of `eth_sendUserOperation`?

Reply:
So this is actually a custom method exported by bundlers to support erc4337 (Smart Wallets). When a user wan't to do a smart wallet transaction, the wallet will call this method on a bundler. Long story short, once submmited to the bundler they will be batched up into a transaction.